### DISCUSSION TITLE
Requesting Function to Delete Extension in ExtensionPay Dashboard

### DISCUSSION THREAD

**Discussion by adamity** [Glench/ExtPay#72] [Ideas]
Hi Everyone! I can't seem to delete the testing extension that I've created in ExtensionPay dashboard, I would like to know if there is a way to delete an extension. Thanks!

![image](https://user-images.githubusercontent.com/50904284/195867175-d746b74f-daeb-4bab-b769-9e64f917da2a.png)

**Comment by ba32107** [comment]
+1 I'm interested in this too!
